src/main/java/core/basesyntax/ForConstants.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public abstract class ForConstants {
```

Comment:
Do you need Application global constants?

Reply:
I moved it into class.